File: app/src/main/java/com/example/myfakequotesapp/Model/Quote.java
package com.example.myfakequotesapp.Model;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Quote {
    @PrimaryKey(autoGenerate = true)
    public int id;
    @ColumnInfo(name="quote_text")
    public String quoteText;
    @ColumnInfo(name = "name_author")
    public String nameAuthor;
    @ColumnInfo(name = "lastname_author")
    public String lastnameAuthor;
    @ColumnInfo(name = "image")
    public String image;
    //@ColumnInfo(name = "title")
    //public String title;
}

File: app/src/main/java/com/example/myfakequotesapp/Model/Quote.kt
package com.example.myfakequotesapp.Model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class Quote {
    @PrimaryKey(autoGenerate = true)
    var id = 0

    @ColumnInfo(name = "quote_text")
    var quoteText: String? = null

    @ColumnInfo(name = "name_author")
    var nameAuthor: String? = null

    @ColumnInfo(name = "lastname_author")
    var lastnameAuthor: String? = null

    @ColumnInfo(name = "image")
    var image: String? = null

    @ColumnInfo(name = "title")
    var title: String? = null
}